Assertion on x(1) != 10 failed on foward analysis.
Searching a trace of failure... found a failing trace!
Trace :
  9 -> 10: x(1) = rand(-100,100)
  10 -> 5: Enter main
  5 -> 8: alpha(2) = 9
  8 -> 1: Enter function f
  1 -> 3: ! (x(1) == alpha(2)) ?
  3 -> 2: x(1) = alpha(2)
  2 -> 7: Exit function f
  Assertion failing point.

